pootle/templates/browser/index.html

```diff
@@ -69,24 +70,28 @@ <h3 class="top">{% trans "Translation Statistics" %}</h3>
             </div>
           </div>
         </div>
+	{% endif %}
       </div>
 
       <div id="js-mnt-top-contributors"></div>
 
       <div class="summary-3-col">
+	{% if stats.last_created_unit %}
         <div id="js-last-updated-wrapper">
           <h3 class="top">{% trans "Updates" %}</h3>
           <div class="bd">
             <div class="graph"></div>
             <div class="action-wrapper">
-              <label>{% trans "Last action:" %}</label>
+              <label>{% trans "Last created:" %}</label>
```

Comment:
Is this only here when a unit is created? What happens if an existing unit changes (or is that just a creation anyway)?